mob.go

```diff
@@ -874,6 +888,9 @@ Basic Commands(Options):
     [--squash]                           Squash commits from wip branch
   reset 
     [--branch|-b <branch-postfix>]       Set wip branch to 'mob/<base-branch>/<branch-postfix>'
+  squash-wip 
```

Comment:
would like this to be in a separate group. Maybe as an experimental feature first until it’s battle tested?

Reply:
like so ?

Reply:
It should no longer be here and part of the basic commands, only in the experimental section.